integrade/tests/api/v1/utils.py

```diff
@@ -27,3 +27,43 @@ def create_user_account(user=None):
     assert json_response['email'] == user['email']
     user.update(response.json())
     return user
+
+
+def get_auth(user=None):
+    """Get authentication for given user to use with requests.
+
+    For example::
+        usr = create_user_account({
+            'email': '',
+            'password': gen_password(),
+            'username': uuid4(),
+        })
+        auth = get_auth(username, pwd)
+        client = api.Client(authenticate=False)
+        client.get(urls.AUTH_ME, auth=auth)
+
+    If no user is provided, a new user is created.
+    This is usefule when you need to make authenticated requests as a regular
+    user, but never need to use the user information for anything else.
+
+    Example::
+
+        from integrade.tests.api.v1 import urls
+        client = api.client(authenticate=False)
+        auth1 = get_user_and_auth()
+        client.get(urls.CLOUD_ACCOUNT, auth=auth1)
+
+    :returns: auth
```

Comment:
Either drop this line or state that the returned obj is an instance of api.TokenAuth